include/ur_api.h

```diff
@@ -963,7 +963,8 @@ urAdapterGetLastError(
 typedef enum ur_adapter_info_t {
     UR_ADAPTER_INFO_BACKEND = 0,         ///< [::ur_adapter_backend_t] Identifies the native backend supported by
                                          ///< the adapter.
-    UR_ADAPTER_INFO_REFERENCE_COUNT = 1, ///< [uint32_t] Reference count of the adapter.
+    UR_ADAPTER_INFO_VERSION = 1,         ///< [uint32_t] Specifies the adapter version, defaults to 1.
```

Comment:
What does "defaults to 1" actually mean here? It seems all adapters set it explicitly.

Reply:
"Default" implies that it is set automatically if you don't set it, which is not the case here.

Perhaps "Starts at 1 and should be incremented on every major adapter release" would be clearer?

Reply:
Updated.